app/views/MainTemplate.scala.html

```diff
@@ -1,74 +1,91 @@
-  timeoutEnabled: Boolean = true,
-  canSignOut: Boolean = true
-)(mainContent: Html)(implicit request : Request[_], messages: Messages)
+    sidebarLinks: Option[Html] = None,
+    contentHeader: Option[Html] = None,
+    bodyClasses: Option[String] = None,
+    mainClass: Option[String] = None,
+    scriptElem: Option[Html] = None,
+    timeoutEnabled: Boolean = true,
+    canSignOut: Boolean = true,
+    showBackLink: Boolean = false
+)(mainContent: Html)(implicit request: Request[_], messages: Messages)
 
-@import uk.gov.hmrc.play.views.html.layouts
-
-@serviceInfoContent = {}
-
-@defaultScriptElem = {
+@headScripts = {
     @if(timeoutEnabled) {
-        <script>
-            window.GOVUK.timeoutDialog({
-            timeout: @appConfig.timeoutLength,
-            countdown: @appConfig.countdownLength,
-            title: '@messages("timeout.title")',
-            message: '@messages("timeout.message")',
-            keepAliveUrl: '@sessionTimeoutController.keepAlive.url',
-            signOutUrl: '@sessionTimeoutController.timeout.url',
-            keepAliveButtonText: '@messages("timeout.keep-alive-button")'
-            });
-        </script>
+        @hmrcTimeoutDialogHelper(
+            signOutUrl = controllers.routes.SessionTimeoutController.timeout.url,
+            keepAliveUrl = Some(controllers.routes.SessionTimeoutController.keepAlive.url),
+            timeout = Some(appConfig.timeoutLength),
+            countdown = Some(appConfig.countdownLength),
+            title = Some(messages("timeout.title")),
+            message = Some(messages("timeout.message")),
+            keepAliveButtonText = Some(messages("timeout.keep-alive-button"))
+        )
     }
-    @scriptElem.getOrElse(HtmlFormat.empty)
+
+    @stylesheets()
 }
 
-@sidebarContent = {
-    @if(sidebarLinks.isDefined) {
-        @sidebar(sidebarLinks.get, Some("sidebar"))
+@beforeContentBlock = {
+    @hmrcLanguageSelectHelper()
+    @if(showBackLink) {
+        @govukBackLink(BackLink(
+            attributes = Map("id" -> "back-link"), classes="js-enabled", href="javascript:history.back()", content = HtmlContent(messages("site.back"))
+        ))
     }
 }
 
-@govuk_wrapper(appConfig = appConfig,
-               title = breadcrumbTitle(title),
-               mainClass = mainClass,
-               bodyClasses = bodyClasses,
-               sidebar = sidebarContent,
-               contentHeader = contentHeader,
-               mainContent = article(mainContent),
-               serviceInfoContent = serviceInfoContent,
-               scriptElem = Some(defaultScriptElem),
-               canSignOut = canSignOut
-)
+@mainContentWithGetHelp = {
+    @mainContent
+    @hmrcReportTechnicalIssueHelper()
+}
+
+@govukLayout(
+    pageTitle = Some(breadcrumbTitle(title)),
+    headBlock = Some(hmrcHead(
+        headBlock = Some(headScripts)
+    )),
+    headerBlock = Some(hmrcStandardHeader(
+        serviceUrl = Some(appConfig.trustsUrl),
```

Comment:
I believe this needs to be `loginContinueUrl` instead of `trustsUrl`

Reply:
Your correct. I missed it. Updated